### PR TITLE
Strings for rocket 1.0

### PR DESCRIPTION
**Pull request by mTwTm**
https://conjuror.github.io/Zerda-Screenshots/index.html

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -699,3 +721,42 @@ msgctxt "remove"
...
+msgctxt "rate_app_dialog_text_title"
+msgid "Love {{app_name}}?"
+msgstr ""
+
+msgctxt "rate_app_dialog_text_content"
+msgid "Thank you for choosing our product. Your feedback is important to us."
+msgstr ""
+
+msgctxt "rate_app_dialog_btn_go_rate"
+msgid "Yes, rate 5 star!"
```
Comment (Delphine) [inline locales/templates/app.pot]: Shouldn't it be "5 stars" here?

File: locales/templates/app.pot [hunk 2]
```diff
@@ -645,6 +657,16 @@ msgctxt "screenshot_image_viewer_dialog_delete_msg"
 msgid "Delete this image?"
 msgstr ""
 
+msgctxt "screenshot_on_boarding_text_msg"
+msgid ""
+"Tap the screenshot button in the menu to capture the whole page.\n"
+"Fast, easy, and come back to the page at anytime!"
```
Comment (Delphine) [inline locales/templates/app.pot]: This sentence sounds really weird. The fact that there are 2 nouns and then a verb is very surprising. It should rather be something like "Fast and easy. Come back to the page at anytime!" or maybe "Fast, easy - come back to the page at anytime!"

File: locales/templates/app.pot (hunk 1 above)
Comment (joechengla) [reply locales/templates/app.pot]: yes :)

File: locales/templates/app.pot (hunk 2 above)
Comment (joechengla) [reply locales/templates/app.pot]: How about let's go with this one?
Fast, easy - come back to the page at anytime!

**Comment by mTwTm** [comment]
Patched! We'll provide the deadline soon in around an hour after our daily meeting :)

**Comment by mTwTm** [comment]
We hope to set the deadline on Friday Taipei time since we're planning to release on that day. Let us know if it's not applicable.

**Comment by Delphine** [comment]
Contacted community, will let you know once they are ready so that we can proceed with screenshot testing! thanks

**Comment by mTwTm** [comment]
Hi Delphine,

I do owe you, the l10n team, and our community a big thank you. As an engineer I should have foreseen the need to have the strings localized, especially when we're launching a product outside of where we based. The Rocket team have failed to anticipate allocating enough time and communication efforts on the localization and with this mindset we understand it is our own action that risks both the schedule and the quality of Rocket. Again I'd like to say thank you for letting us ride the train and give Rocket a try to see if it has the luck to land on time. I'll keep an eye on having this discussed in our retrospective meetings and I'll make every effort to make sure this doesn't happen again.

**Comment by Delphine** [comment]
Thanks - let's coordinate better next time. I'm available on slack (sometimes) and irc (always, #mobile channel for example) and can participate in some cross-functional meetings when they are PST-friendly (I can be flexible about this sometimes, as long as it's still a reasonable time ;) ).

**Pull request merged**